src/apipkg/__init__.py

```diff
@@ -180,6 +180,24 @@ def __dict__(self):
         return dict
 
 
+def _py_test_hack(modpath, attrname, importerror_alternative):
```

Comment:
Probably good to leave a docstring here explaining what is the hack and why it is necessary.

Reply:
Actually I noticed this is not being used anywhere... is that intentional?

Reply:
missed code drop from the simplification

Reply:
removed